PWGCF/FEMTOSCOPY/FemtoDream/AliAnalysisTaskPOmegaPenne.cxx

```diff
@@ -30,31 +30,31 @@ ClassImp(AliAnalysisTaskPOmegaPenne)
                                                                 VTrack(0),
                                                                 fEvent(0),
                                                                 fTrack(0),
+                                                                fConfig(0),
+                                                                fGTI(0),
+                                                                fTrackBufferSize(5000),
                                                                 fEventCuts(0),
-                                                                fEventCuts2(0),
                                                                 fv0(0),
-                                                                fv0_2(0),
                                                                 fCascade(0),
```

Comment:
I remember @a-mathis fixed some compiler warnings - are you reverting his fixes now?

Reply:
Should actually get less warnings. I got warnings for not in right order initialization, which is fixed here

Reply:
aye, I didn't fix ordering in user tasks
thanks a lot @magicep0t for taking care of that!